website/docs/components/tabs/partials/guidelines/guidelines.md

```diff
@@ -47,6 +47,21 @@ Don’t trigger a Tab change via an external action, such as a submit or next bu
 ![Tabs with external trigger](/assets/components/tabs/tabs-external-trigger.png)
 !!!
 
+## Sizes
+
+Medium-sized tabs work best in most contexts. Large tabs can be used to emphasize important sections, enhance clarity, and convey hierarchical distinction. 
```

Comment:
```suggestion
Medium tabs are the default and work best in most contexts. Large tabs can be used to emphasize important sections, enhance clarity, and convey hierarchical distinction. 
```

One other note; I think you can remove this for consistency with the rest of the language. Also, per @KristinLBradley 's recommendation, could add in a note about the default here.